bitnami/influxdb/templates/deployment.yaml

```diff
@@ -73,8 +73,10 @@ spec:
               find /bitnami/influxdb/{data,meta,wal} -mindepth 0 -maxdepth 1 | \
               {{- if eq ( toString ( .Values.volumePermissions.securityContext.runAsUser )) "auto" }}
                 xargs -r chown -R `id -u`:`id -G | cut -d " " -f2`
+              chown `id -u`:`id -G | cut -d " " -f2` /bitnami/influxdb
```

Comment:
I think that instead of adding a new line for change the ownership for `/bitnami/influxdb` you could add `.` to the directories list in line 73 (`find /bitnami/influxdb/{....}`).
 
What do you think ?

Reply:
I think it is good idea. Simplified code, please review.